### PR TITLE
Updated privacy page

### PR DESCRIPTION
**Pull request by EphraimAssi**
# Description

[374 Privacy Policy Page EN/FR](https://trello.com/c/mty2DKLm/374-374-privacy-policy-page-en-fr)

I updated the privacy policy page to match the figma design. I also updated the menu bar as well.

## Acceptance Criteria

The policy page needs to look like the figma design. The menu bar must be updated as well.

## Test Instructions

1. Run the code and go on the policy page and compare it to the figma design
2. Make sure the menu and the it's links work

## Checklist

- [ ] Strings use placeholders for translation (No hard coded strings)
- [ ] Unit tests have been added/updated

## Product and Sprint Backlog

[Trello](https://trello.com/b/ZqWtJSyh/alpha-site-board)

### CODE REVIEW

File: public/locales/en/common.json [hunk 1]
```diff
@@ -253,5 +252,9 @@
   "sentEmailTo": "We have sent an email to",
   "toCheckEmail": "for you to verify and confirm the email address you provided.",
   "pleaseCheckYourEmail": "Please check your emails (Step 2 of 3)",
-  "unsubscribeConfirmation": "Please click the confirmation link in the email within 24 hours to finish unsubscribing."
+  "unsubscribeConfirmation": "Please click the confirmation link in the email within 24 hours to finish unsubscribing.",
+  "withdraw": "How to withdraw your information",
```
Comment (var-kyle) [inline public/locales/en/common.json:256]: The `withdraw` key was missed in the french file
image

**Review by var-kyle — CHANGES_REQUESTED** [review]
The page looks great, just missed a key in the french translation file.

I'm also not sure if this is being taken care of in another PR, but the text in the button calling for a signup is off between here and the Figma designs. You can ignore this if it's being taken care of somewhere else.
this pr:
image

Figma:
image

**Comment by EphraimAssi** [comment]
I also added the "unsubscribeConfirmation" key for french translation since it wasn't there

**Comment by nibivi77** [comment]
> The page looks great, just missed a key in the french translation file.
> 
> I'm also not sure if this is being taken care of in another PR, but the text in the button calling for a signup is off between here and the Figma designs. You can ignore this if it's being taken care of somewhere else.
> this pr:
> image
> 
> Figma:
> image

I'll take care of the

**Comment by nibivi77** [comment]
Sorry I closed it by mistake.

**Review by nibivi77 — APPROVED** [review]
Nice ! Good job!

**Review by var-kyle — APPROVED** [review]
Looks great! Nice work!

**Pull request merged**